tailwind.config.js

```diff
@@ -1,33 +1,23 @@
-const plugin = require('tailwindcss/plugin')
+const plugin = require("tailwindcss/plugin");
```

Comment:
```suggestion
/** @type {import('tailwindcss').Config} */

const plugin = require("tailwindcss/plugin");
```

This is a new thing since ~v3 that gives some type checking in this config file even though it is a .js file. Ideally, please check it once locally.